code/sound/sound.cpp

```diff
@@ -354,13 +358,15 @@ sound_load_id snd_load(game_snd_entry* entry, int flags, int /*allow_hardware_lo
 	std::unique_ptr<sound::IAudioFile> audio_file = openAudioFile(entry->filename);
 
 	if (audio_file == nullptr) {
+		if (flags)
+			*flags |= GAME_SND_NOT_VALID;
 		return sound_load_id::invalid();
```

Comment:
Probably out of scope, but should there be an `nprintf()`, like there is below, that creating the audio file object failed?

Reply:
There is, but it's deeper down in `audio_file->Open`.